Registration/registrationServices.py

```
@@ -0,0 +1,7 @@
+import requests
+import os
+class RegistrationServices:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)